src/v2i-hub/CDASimAdapter/src/CDASimConnection.cpp

```diff
@@ -193,6 +193,8 @@ namespace CDASimAdapter{
 
     void CDASimConnection::forward_message( const std::string &msg, const std::shared_ptr<UdpClient> _client ) const {
         if ( !msg.empty() && _client) {
+            PLOG(logDEBUG) << "Sending UDP msg " << msg << " to host " << _client->GetAddress() 
```

Comment:
Is msg is empty, do we want to log a warning or debug statement as well for later debug purpose?

Reply:
Good Idea. Will add that.